### ISSUE TITLE
[5.x.x] Ion-checkbox accessibility broken

### ISSUE THREAD

**Issue by NessWork** [ionic-team/ionic-native#3176]
[x ] bug report

**Current behavior:**
Ion-checkbox's button does not have an accessible name.

**Expected behavior:**
Ion-checkbox should allow a prop named aria-label and set it on Ion-checkbox's internal button.

**Steps to reproduce:**
Use Ion-checkbox in a simple html and run Lighthouse Audit from Chrome.
It shows the following error for accessibility :

Buttons do not have an accessible name
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

**Related code:**
```
 <ion-checkbox></ion-checkbox>
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Ionic:

   Ionic CLI                     : 5.2.0
   Ionic Framework               : @ionic/angular 4.6.2
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

System:

   NodeJS : v10.13.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.4.1
   OS     : Windows 10
```
